spec/features/admin/view_organisation_tab_spec.rb

```diff
@@ -12,11 +12,11 @@
     let(:organisation) { create(:organisation) }
     let(:user) { create(:user, :confirmed, email: 'me@example.gov.uk', organisation: organisation, admin: true) }
 
-    it 'displays the organisation tab' do
+    it 'displays not the organisation tab' do
```

Comment:
This reads weirdly to me.

```suggestion
    it 'does not display the organisation tab' do
```

Reply:
Yeah, forgot to read over 😁